### PR TITLE
Multiple accept-pulls

### PR DESCRIPTION
**Pull request by chrisarcand**
Allows for multiple PR IDs to be used in `accept-pull`

Instead of this:

```plaintext
$ op accept-pull 123
$ op accept-pull 456
$ op accept-pull 789
```

You can do this:

```plaintext
$ op accept-pull 123 456 789
```

As this change is a trivial one in the GLI interface, no tests are provided (testing the GLI interface is not something we do)

If you wish to QA this change, using RVM:
* Create a new rvm gemset in a project that has PRs to merge
* Pull down this branch, build the gem.
* gem install the built gem to the empty gemset
* `which op` to ensure your `op` bin is pointing to your built gemset
* Run it!

Screenshot:

### CODE REVIEW

**Comment by ChrisArcand** [comment]
Talking with @anfleene, I'll add some sort of check with helpful message to abort if you get some sort of merge conflict halfway through.

File: lib/octopolo/scripts/accept_pull.rb [hunk 1]
```diff
@@ -11,56 +11,60 @@ class AcceptPull
...
       def execute
         GitHub.connect do
-          pull_request = GitHub::PullRequest.new(config.github_repo, pull_request_id)
-          merge pull_request
-          update_changelog pull_request
-        end
-      rescue GitHub::PullRequest::NotFound
-        cli.say "Unable to find a pull request #{pull_request_id} for #{config.github_repo}. Please verify."
-      end
+          Git.fetch
```
Comment (NickLaMuro) [inline lib/octopolo/scripts/accept_pull.rb]: Spacing

File: lib/octopolo/scripts/accept_pull.rb [hunk 2]
```diff
@@ -11,56 +11,60 @@ class AcceptPull
...
       include CLIWrapper
 
-      attr_accessor :pull_request_id
+      attr_accessor :pull_request_ids
 
-      def self.execute(pull_request_id)
-        pull_request_id ||= Integer(cli.prompt "Pull Request ID: ")
-        new(pull_request_id).execute
+      def self.execute(pull_request_ids)
+        pull_request_ids ||= Integer(cli.prompt "Pull Request ID: ")
```
Comment (anfleene) [inline lib/octopolo/scripts/accept_pull.rb]: add an `s` to `Pull Request ID:`

**Comment by anfleene** [comment]
I think this looks good other than the one missing `s` I'm a little concerned about what happens when a PR fails to merge in the middle of your list. Will we know what has been merged and what hasn't. It should be too hard since we update the changelog after the merge it keeps a pretty good record. Also the    git log will tell us, so I think it's fine.

Have you done any QA yet?

**Comment by ChrisArcand** [comment]
There's a couple little tweaks that need doing quite yet (found from my own QA). I'll be doing a bit more work on it and let you know when it's ready again.

I've also been mulling your same thoughts on 'how reversible should this be'?

Debating between: 

"This command could be smarter (and more complex) by going through the trouble of saving SHAs and doing a hard reset of everything in the case of a problem in the PR chain" 

and 

"This is an interface to Git. If you know how to Git (and we do), don't make things needlessly complex. Allow conflicts to occur in a normal Git way and the user can deal with it as they so choose. And it's not like it's difficult to just hard reset back to the pre-merge state if you want to bail (and you won't have pushed to the remote at that time, either)."

Heavily leaning towards the latter, especially if I make things a bit more informative by showing which PRs were merged to the conflict point and which ones weren't.

**Comment by anfleene** [comment]
I agree, lets error on the side of simplicity

**Comment by chrisarcand** [comment]
Stale, closing...

**Pull request closed without merging**